### PR TITLE
Address a public review feedback

### PR DESCRIPTION
**Pull request by Adam-pi3**
1) Clarify PM in regards to DMA (address #69)
2) Clarify PM in regards to hypervisor load/stores when
   MPRV=1 and MXR=1 (address #70)
3) add language stating that pointer masking with HLV.* and HLS.*
   applies to VS mode as well, in addition to U mode (address #72)
4) Fix Makefile to generate pdf (image source files has been changed)

### CODE REVIEW

File: zjpm/background.adoc [hunk 1]
```diff
@@ -126,7 +126,7 @@ This list will grow over time as new extensions introduce new instructions that
 
 For other extensions, pointer masking applies to all explicit memory accesses by default. Future extensions may add specific language to indicate whether particular accesses are or are not included in pointer masking.
 
-MPRV and SPVP affect pointer masking as well, causing the pointer masking settings of the effective privilege mode to be applied. The exception to this rule is the case where MPRV is set while MXR is also set, in which case pointer masking does not apply (see <<_ssnpm>> for the rationale).
+MPRV and SPVP affect pointer masking as well, causing the pointer masking settings of the effective privilege mode to be applied. The exception to this rule is the case where MPRV is set while MXR is also set, in which case pointer masking does not apply (see <<_ssnpm>> for the rationale). As a result, hypervisor load/stores are not affected by the MPRV=1 and MXR=1 exception. For 2-stage translation, whether or not pointer masking applies is determined by mstatus.MXR as opposed to vsstatus.MXR.
```
Comment (martinmaas) [inline zjpm/background.adoc:129]: I would probably remove "As a result", as the sentence does not depend on any of the preceding ones.

File: zjpm/instructions.adoc [hunk 2]
```diff
@@ -25,7 +25,7 @@ All of these fields are read-only 0 on RV32 systems.
...
+In systems where the H Extension is present, `Ssnpm` also adds a new 2-bit WARL field (`PMM`) to bits 33:32 of `henvcfg`. Setting `PMM` enables or disables pointer masking for VS-mode, according to the values in Table 2. Further, a 2-bit WARL field (`HUPMM`) is added to bits 49:48 of `hstatus`. Setting `hstatus.HUPMM` enables or disables pointer masking for `HLV.\*` and `HSV.*` instructions in U-mode, according to the values in Table 2, when their explicit memory access is performed as though in VU-mode or as though in VS mode when SPVP is 1. In HS- and M-modes, pointer masking for these instructions is enabled or disabled by `senvcfg.PMM`, when their explicit memory access is performed as though in VU-mode. Setting `henvcfg.PMM` enables or disables pointer masking for `HLV.\*` and `HSV.*` when their explicit memory access is performed as though in VS-mode.
```
Comment (martinmaas) [inline zjpm/instructions.adoc:28]: Nit: "VS-mode" for consistency.

File: zjpm/background.adoc [hunk 3]
```diff
@@ -126,7 +126,12 @@ This list will grow over time as new extensions introduce new instructions that
 
 For other extensions, pointer masking applies to all explicit memory accesses by default. Future extensions may add specific language to indicate whether particular accesses are or are not included in pointer masking.
 
-MPRV and SPVP affect pointer masking as well, causing the pointer masking settings of the effective privilege mode to be applied. The exception to this rule is the case where MPRV is set while MXR is also set, in which case pointer masking does not apply (see <<_ssnpm>> for the rationale).
+[NOTE]
+====
+It is worth noting that Pointer Masking is not applied to SFENCE.\*, HFENCE.*, SINVAL.\*, or HINVAL.*. When such operation is invoked, it is the responsibility of software to provide the correct address.
```
Comment (ved-rivos) [inline zjpm/background.adoc:131]:
```suggestion
It is worth noting that pointer masking is not applied to SFENCE.\*, HFENCE.*, SINVAL.\*, or HINVAL.*. When such an operation is invoked, it is the responsibility of the software to provide the correct address.
```

**Comment by Adam-pi3** [comment]
Thanks @ved-rivos

**Pull request merged**